### PR TITLE
refactor(egen): Refactored custom batch prediction feature filter code according to template guidelines

### PR DESCRIPTION
**Pull request by rohith-egen**
Refactored all the cells according to the template notebook
<br>
List of updates:
1. Steps are re-organized based on the [new template](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/notebook_template.ipynb).
2. Renamed REGION as LOCATION
3. Removed import os - unused
4. Included colab enterprise link 
5. Used generalized code for "colab only" cells
<br><br><br>

**REQUIRED:** Fill out the below checklists or remove if irrelevant
1. If you are opening a PR for `Official Notebooks` under the [notebooks/official](https://github.com/GoogleCloudPlatform/vertex-ai-samples/tree/main/notebooks/official) folder, follow this mandatory checklist:
- [x] Use the [notebook template](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/notebook_template.ipynb) as a starting point.
- [x] Follow the style and grammar rules outlined in the above notebook template.
- [x] Verify the notebook runs successfully in Colab since the automated tests cannot guarantee this even when it passes.
- [x] Passes all the required automated checks. You can locally test for formatting and linting with these [instructions](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/CONTRIBUTING.md#code-quality-checks).
- [ ] You have consulted with a tech writer to see if tech writer review is necessary. If so, the notebook has been reviewed by a tech writer, and they have approved it.
- [ ] This notebook has been added to the [CODEOWNERS](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/official/CODEOWNERS) file under the `Official Notebooks` section, pointing to the author or the author's team.
- [x] The Jupyter notebook cleans up any artifacts it has created (datasets, ML models, endpoints, etc) so as not to eat up unnecessary resources.

<br>

### CODE REVIEW

File: notebooks/official/prediction/custom_batch_prediction_feature_filter.ipynb [hunk 1]
```diff
@@ -30,39 +30,30 @@
...
+        "      <img src=\"https://cloud.google.com/ml-engine/images/colab-logo-32px.png\" alt=\"Google Colaboratory logo\"><br> Open in Colab\n",
         "    </a>\n",
         "  </td>\n",
-        "  <td>\n",
-        "    <a href=\"https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/official/prediction/custom_batch_prediction_feature_filter.ipynb\">\n",
-        "      <img src=\"https://cloud.google.com/ml-engine/images/github-logo-32px.png\" alt=\"GitHub logo\">\n",
-        "      View on GitHub\n",
+        "  <td style=\"text-align: center\">\n",
+        "    <a href=\"https://console.cloud.google.com/vertex-ai/colab/import/https:%2F%2Fraw.githubusercontent.com%2FGoogleCloudPlatform%2Fvertex-ai-samples%2Fmain%2Fnotebooks%2Fofficial%2Fprediction%2Fcustom_batch_prediction_feature_filter.ipynb\">\n",
+        "      <img width=\"32px\" src=\"https://lh3.googleusercontent.com/JmcxdQi-qOpctIvWKgPtrzZdJJK-J3sWE1RsfjZNwshCFgE_9fULcNpuXYTilIR2hjwN\" alt=\"Google Cloud Colab Enterprise logo\"><br> Open in Colab Enterprise\n",
```
Comment (kittyabs) [inline notebooks/official/prediction/custom_batch_prediction_feature_filter.ipynb:41]: line 41: change logo URL to:
- https://cloud.google.com/ml-engine/images/colab-enterprise-logo-32px.png

File: notebooks/official/prediction/custom_batch_prediction_feature_filter.ipynb [hunk 2]
```diff
@@ -134,15 +125,22 @@
...
     {
       "cell_type": "markdown",
       "metadata": {
         "id": "install_aip"
       },
       "source": [
-        "## Installation\n",
-        "\n",
-        "Install the following packages required to execute this notebook. "
+        "### Install Vertex AI SDK and other required packages"
```
Comment (kittyabs) [inline notebooks/official/prediction/custom_batch_prediction_feature_filter.ipynb:143]: change to: 

- Install Vertex AI SDK for Python and other required packages

File: notebooks/official/prediction/custom_batch_prediction_feature_filter.ipynb [hunk 3]
```diff
@@ -379,48 +278,52 @@
     {
       "cell_type": "markdown",
       "metadata": {
-        "id": "import_aip"
+        "id": "6a870411c189"
       },
       "source": [
-        "### Import libraries"
+        "### Set Google Cloud project information and initialize Vertex AI SDK\n",
```
Comment (kittyabs) [inline notebooks/official/prediction/custom_batch_prediction_feature_filter.ipynb:284]: change to: "Vertex AI SDK for Python"

File: notebooks/official/prediction/custom_batch_prediction_feature_filter.ipynb [hunk 4]
```diff
@@ -570,7 +473,7 @@
         "id": "5c7732822757"
       },
       "source": [
-        "## Create a Vertex AI tabular Dataset from BigQuery dataset\n",
```
Comment (kittyabs) [inline notebooks/official/prediction/custom_batch_prediction_feature_filter.ipynb:573]: change "Dataset" to lower case:

- Create a Vertex AI tabular dataset ...

Comment (kittyabs) [reply notebooks/official/prediction/custom_batch_prediction_feature_filter.ipynb:573]: line 476

File: notebooks/official/prediction/custom_batch_prediction_feature_filter.ipynb [hunk 5]
```diff
@@ -134,15 +125,22 @@
         "to generate a cost estimate based on your projected usage."
       ]
     },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "3b1ffd5ab768"
+      },
+      "source": [
+        "## Getting Started"
```
Comment (kittyabs) [inline notebooks/official/prediction/custom_batch_prediction_feature_filter.ipynb:134]: change to "Get started"

File: notebooks/official/prediction/custom_batch_prediction_feature_filter.ipynb [hunk 6]
```diff
@@ -158,174 +156,75 @@
...
-        "id": "wsePm9c4jmpT"
-      },
-      "outputs": [],
-      "source": [
-        "PROJECT_ID = \"[your-project-id]\"  # @param {type:\"string\"}\n",
-        "\n",
-        "# Set the project id\n",
-        "! gcloud config set project {PROJECT_ID}"
+        "<div class=\"alert alert-block alert-warning\">\n",
+        "<b>⚠️ The kernel is going to restart. Please wait until it is finished before continuing to the next step. ⚠️</b>\n",
```
Comment (kittyabs) [inline notebooks/official/prediction/custom_batch_prediction_feature_filter.ipynb:198]: Delete "Please" and contract "it is"

- Wait until it's finished before...

**Review by katiemn — APPROVED** [review]

**Pull request merged**